src/game-engine/GameEngineB.ts

```diff
@@ -10,9 +9,10 @@ import Propose from './positions/Propose';
 import Accept from './positions/Accept';
 import Resting from './positions/Resting';
 import Conclude from './positions/Conclude';
+import { Wallet } from '../wallet';
```

Comment:
I'm in two minds about whether we should keep `ChannelWallet` instead of `Wallet` to avoid confusion with things like meta-mask. This is an easy change to make later though, so happy with this here.

Reply:
I agree `Wallet` is pretty generic, would be nice to have a better name